src/coupling/hm_iterative.cc

```diff
@@ -102,6 +109,9 @@ void HM_Iterative::EqData::initialize(Mesh &mesh)
     
     potential_ptr_ = create_field_fe<3, FieldValue<3>::Scalar>(mesh, MixedPtr<FE_CR>());
     pressure_potential.set(potential_ptr_, 0.0);
+
+    ref_potential_ptr_ = create_field_fe<3, FieldValue<3>::Scalar>(mesh, MixedPtr<FE_CR>());
+    ref_pressure_potential.set(ref_potential_ptr_, 0.0);
```

Comment:
What prevents passing a copy of flow bc_pressure to elasticity? We could possibly add setting to zero for the Neumann boundary condition. It should also work for Seepage and River BC in Darcy.

Reply:
The reason is to minimize the interface between coupled processes. Mechanics should not be spoiled by external fields from flow or other equations.

Reply:
OK. So copy the comments to a new issue and merge this branch. In a separate branch we can then test what exactly prevents:
1. usage of copy of bc_pressure (without .value call)
2. having FieldModel of the boundary field ref_pressure as a copy of bc_pressure